bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/handler/EspMilightHubHandler.java

```diff
@@ -0,0 +1,385 @@
+                    && "rgbw".equals(globeType)) {
+                sendMQTT("{\"command\":\"set_white\"}");
+                return;
+            }
+            sendMQTT("{\"state\":\"ON\",\"level\":" + hsb.getBrightness().intValue() + ",\"hue\":"
+                    + hsb.getHue().intValue() + ",\"saturation\":" + hsb.getSaturation().intValue() + "}");
+            savedLevel = hsb.getBrightness().toBigDecimal();
+            return;
+        } else if (command instanceof PercentType) {
+            PercentType percentType = (PercentType) command;
+            if (percentType.intValue() == 0) {
+                turnOff();
+                return;
+            } else if (percentType.intValue() == 1 && config.oneTriggersNightMode) {
+                sendMQTT("{\"command\":\"night_mode\"}");
+                return;
+            }
+            sendMQTT("{\"state\":\"ON\",\"level\":" + command + "}");
+            savedLevel = percentType.toBigDecimal();
+            if (globeType.equals("rgb_cct") || globeType.equals("fut089")) {
+                if (config.dimmedCT > 0 && bulbMode.equals("white")) {
+                    sendMQTT("{\"state\":\"ON\",\"color_temp\":" + autoColourTemp(savedLevel.intValue()) + "}");
+                }
+            }
+            return;
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            return;
+        }
+        switch (channelUID.getId()) {
+            case CHANNEL_LEVEL:
+                handleLevelColour(command);
+                return;
+            case CHANNEL_BULB_MODE:
+                bulbMode = command.toString();
+                break;
+            case CHANNEL_COLOURTEMP:
+                int scaledCommand = (int) Math.round((370 - (2.17 * Float.valueOf(command.toString()))));
+                sendMQTT("{\"state\":\"ON\",\"level\":" + savedLevel + ",\"color_temp\":" + scaledCommand + "}");
+                break;
+            case CHANNEL_COMMAND:
+                sendMQTT("{\"command\":\"" + command + "\"}");
+                break;
+            case CHANNEL_DISCO_MODE:
+                sendMQTT("{\"mode\":\"" + command + "\"}");
+                break;
+            case CHANNEL_COLOUR:
+                handleLevelColour(command);
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(ConfigOptions.class);
+        if (config.dimmedCT > 0) {
+            maxColourTemp = new BigDecimal(config.favouriteWhite);
+            minColourTemp = new BigDecimal(config.dimmedCT);
+            if (minColourTemp.intValue() <= maxColourTemp.intValue()) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
+                        "The dimmedCT config value must be greater than the favouriteWhite value.");
+            }
```

Comment:
Please use the correct ThingStatusDetail, also please return immediately when there is a config error.
```suggestion
                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
                        "The dimmedCT config value must be greater than the favourite White value.");
                return;
            }
```